src/acvp_kdf108.c

```diff
@@ -104,16 +122,33 @@ static ACVP_RESULT acvp_kdf108_init_tc (ACVP_CTX *ctx,
     rv = acvp_hexstr_to_bin(key_in, stc->key_in, key_in_len, NULL);
     if (rv != ACVP_SUCCESS) return rv;
 
-    // Allocate space for the key_out
+    if (iv != NULL) {
+        /*
+         * Feedback mode.
```

Comment:
Should we double check mode here too?

Reply:
Don't think we need to since the other function does the check (which you already know) :) If the design changes in the future we can adapt as needed